projects/plugins/jetpack/changelog/-fix-add-ts-vs-code-support

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: fix
```

Comment:
```suggestion
Type: other
```

I'm not sure how you're generating these. If you use `jetpack changelog add` before committing (or `jetpack changelog add [project]`, e.g. `jetpack changelog add plugins/jetpack`) it'll interactively ensure a compliant changelog file for the project's parameters.

Reply:
Oh, I didn't know we could use `jetpack changelog add`, TIL

Thank you